exams/management/commands/authorization_user_exam.py

```diff
@@ -48,25 +37,9 @@ def handle(self, *args, **options):
         username = options.get("username", None)
         course_key = options.get("course_key", None)
 
-        programs = Program.objects.filter(exam_series_code__isnull=False, live=True).filter(~Q(exam_series_code=""))
-        if program_id:
-            programs = programs.filter(id=program_id)
-
-        for program in programs:
-            users_qset = ProgramEnrollment.objects.filter(program=program)
-            if username:
-                users_qset = users_qset.filter(user__username=username)
-
-            users = users_qset.values_list('user', flat=True)
-
-            for user in users:
-                enrollments_qset = CachedEnrollment.user_course_qset(user, program=program)
-                if course_key:
-                    enrollments_qset = enrollments_qset.filter(course_run__edx_course_key=course_key)
-
-                mmtrack = get_mmtrack(user, program)
+        bulk_authorize_for_exam(
+            program_id=program_id,
+            username=username
+        )
 
-                for enrollment in enrollments_qset:
-                    # if user has passed and paid for the course
-                    # and not already authorized for exam the create authorizations.
-                    authorize_for_exam(mmtrack, enrollment.course_run)
+        print("Done with authorization process")
```

Comment:
please do not use "print" statements: this is an example of how to proper write output of management commands (also with a nice output) taken from
https://docs.djangoproject.com/en/1.10/howto/custom-management-commands/
```
self.stdout.write(self.style.SUCCESS('Successfully closed poll "%s"' % poll_id))
```